### ISSUE TITLE
Inconsistent SQL results on property names with dots

### ISSUE THREAD

**Issue by sboucard** [orientechnologies/orientdb#7916]
#### OS: Windows 10 

I created an element with a property whose name contains a dot from Java using following code:
```
        OElement element = session.newElement("Test");
        element.setProperty("with.dot", "hello");
        element.save();
        session.commit();
```

Executing a SQL `SELECT * from Test` from Java returns an OElement which has the property value correctly assigned:
```
        OResultSet results = session.query("SELECT * from Test");
        while (results.hasNext()) {
            OElement element = results.next().toElement();
            system.out.println(element);
```
displays:
> Test#21:0{with.dot:hello} v1

Executing the very same SQL query from Studio or Console shows the element but the property "with.dot" is not set (value is empty).

It seems I get different results for the same SQL query depending on where I do execute it. Is there a known limitation here?

Moreover, the 3.0.0m2 release notes (http://orientdb.com/docs/3.0.x/release/3.0/What-is-new-in-OrientDB-3.0.html) claim this:

> Property names can now contain any character, including blank spaces, dots, brackets and special characters.

Is there a setting I should put somewhere to allow SQL to correctly retrieve the dot property value?

**Comment by luigidellaquila** [comment]
Hi @sboucard 

I'd say it's a bug in the http protocol or in the rendering, I'll check it asap

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @sboucard 

I just checked it and fixed the root cause. I'm running the tests now, if everything is OK I'll push the fix in next few hours

Thanks

Luigi

**Comment by luigidellaquila** [comment]
Hi @sboucard 

I just pushed a fix on develop branch, it will be released with 3.0RC1

Thanks

Luigi
